### PR TITLE
docs: add kubernetes version downgrade warning and hello-universe pack version DOC-1165 DOC-1174

### PR DESCRIPTION
**Pull request by addetz**
## Describe the Change

This patch makes the following changes:
- Adds Kubernetes downgrade warning to cluster update pages and Kubernetes packs
- Adds version 1.1.2 to the hello-universe pack

## Changed Pages

💻 [Kubernetes Upgrades](https://deploy-preview-2786--docs-spectrocloud.netlify.app/integrations/kubernetes-support/#kubernetes-upgrades)
💻 [Cluster Updates](https://deploy-preview-2786--docs-spectrocloud.netlify.app/clusters/cluster-management/cluster-updates/)
💻 [Kubernetes Generic](https://deploy-preview-2786--docs-spectrocloud.netlify.app/integrations/kubernetes-generic/)
💻 [Kubernetes eXtended](https://deploy-preview-2786--docs-spectrocloud.netlify.app/integrations/kubernetes/)
💻 [Kubernetes eXtended edge](https://deploy-preview-2786--docs-spectrocloud.netlify.app/integrations/kubernetes-edge/)
💻 [Hello Universe](https://deploy-preview-2786--docs-spectrocloud.netlify.app/integrations/hello-universe/)

## Jira Tickets

🎫 [DOC-1165](https://spectrocloud.atlassian.net/browse/DOC-1165)
🎫 [DOC-1174](https://spectrocloud.atlassian.net/browse/DOC-1174)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1165]: https://spectrocloud.atlassian.net/browse/DOC-1165?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOC-1174]: https://spectrocloud.atlassian.net/browse/DOC-1174?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/integrations/hello-universe.md [hunk 1]
```diff
@@ -11,12 +11,28 @@ tags: ["packs", "hello-universe", "app-services"]
...
 [Hello Universe](https://github.com/spectrocloud/hello-universe) is a demo web application utilized to help users learn
-more about [Palette](../introduction/introduction.md) and its features. It functions as a standalone front-end
-application and provides users with a local click counter and funny Spectro Cloud-themed images.
+more about [Palette](https://docs.spectrocloud.com/introduction) and its features.
+
+You can deploy it using two preset configurations:
+
+- A standalone front-end application. It provides a click counter that is saved locally and displays Spectro Cloud
+  themed images.
+- A three-tier application with a front-end application, API server and PostgreSQL database into a Kubernetes cluster.
```
Comment (caroldelwing) [inline docs/docs-content/integrations/hello-universe.md:20]:
```suggestion
- A three-tier application with a front-end tier, API server, and PostgreSQL database.
```
maybe? :)

File: docs/docs-content/integrations/hello-universe.md [hunk 2]
```diff
@@ -11,12 +11,28 @@ tags: ["packs", "hello-universe", "app-services"]
...
+  It provides a click counter that is saved in the deployed database and displays Spectro Cloud themed images. You can
+  read more about this configuration on the Hello Universe
+  [README](https://github.com/spectrocloud/hello-universe?tab=readme-ov-file#reverse-proxy-with-kubernetes).
+
+:::info
+
+The three-tier application mode is only supported by version 1.1.2.
```
Comment (caroldelwing) [inline docs/docs-content/integrations/hello-universe.md:27]:
```suggestion
The three-tier application configuration is only supported by version 1.1.2 of the pack.
```

File: docs/docs-content/integrations/hello-universe.md [hunk 3]
```diff
@@ -74,6 +90,86 @@ data "spectrocloud_pack" "hellouniverse" {
...
+
+- Ensure sufficient CPU resources within the cluster to allocate a minimum of 500 milliCPU and a maximum of 500 milliCPU
+  per replica.
+
+## Parameters
+
+The following parameters are applied to the **hello-universe.yaml** manifest through the **values.yaml** file. Users do
+not need to take any additional actions regarding these parameters.
+
+| **Parameter**                       | **Description**                                                                                                                                  | **Default Value**                                                                              | **Required** |
```
Comment (caroldelwing) [inline docs/docs-content/integrations/hello-universe.md:117]: the table looks broken :(

File: docs/docs-content/integrations/hello-universe.md [hunk 4]
```diff
@@ -74,6 +90,86 @@ data "spectrocloud_pack" "hellouniverse" {
...
+| ----------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------- | ------------ | --- | --- |
+| `manifests.namespace`               | The namespace in which the application will be deployed.                                                                                         | `hello-universe`                                                                               | Yes          |
+| `manifests.images.hellouniverse`    | The [`hello-universe`](https://github.com/spectrocloud/hello-universe) application image that will be utilized to create the containers.         | `ghcr.io/spectrocloud/hello-universe:1.1.2`/ `ghcr.io/spectrocloud/hello-universe:1.1.2-proxy` | Yes          |
```
Comment (caroldelwing) [inline docs/docs-content/integrations/hello-universe.md:120]:
```suggestion
| `manifests.images.hellouniverse`    | The [`hello-universe`](https://github.com/spectrocloud/hello-universe) application image that will be utilized to create the front-end containers.         | `ghcr.io/spectrocloud/hello-universe:1.1.2`/ `ghcr.io/spectrocloud/hello-universe:1.1.2-proxy` | Yes          |
```

File: docs/docs-content/integrations/hello-universe.md [hunk 5]
```diff
@@ -74,6 +90,86 @@ data "spectrocloud_pack" "hellouniverse" {
...
+| `manifests.images.hellouniverseapi` | The [`hello-universe-api`](https://github.com/spectrocloud/hello-universe-api) application image that will be utilized to create the containers. | `ghcr.io/spectrocloud/hello-universe-api:1.0.12`                                               | No           |
```
Comment (caroldelwing) [inline docs/docs-content/integrations/hello-universe.md:121]:
```suggestion
| `manifests.images.hellouniverseapi` | The [`hello-universe-api`](https://github.com/spectrocloud/hello-universe-api) application image that will be utilized to create the API containers. | `ghcr.io/spectrocloud/hello-universe-api:1.0.12`                                               | No           |
```

File: docs/docs-content/integrations/hello-universe.md [hunk 6]
```diff
@@ -74,6 +90,86 @@ data "spectrocloud_pack" "hellouniverse" {
...
+| `manifests.images.hellouniversedb`  | The [`hello-universe-db`](https://github.com/spectrocloud/hello-universe-db) application image that will be utilized to create the containers.   | `ghcr.io/spectrocloud/hello-universe-db:1.0.2`                                                 | No           |
```
Comment (caroldelwing) [inline docs/docs-content/integrations/hello-universe.md:122]:
```suggestion
| `manifests.images.hellouniversedb`  | The [`hello-universe-db`](https://github.com/spectrocloud/hello-universe-db) application image that will be utilized to create the database containers.   | `ghcr.io/spectrocloud/hello-universe-db:1.0.2`                                                 | No           |
```

**Review by caroldelwing — APPROVED** [review]
Approved with tiny comments :rocket:

File: docs/docs-content/integrations/hello-universe.md (hunk 3 above)
Comment (addetz) [reply docs/docs-content/integrations/hello-universe.md]: Fixed! Thanks a lot ❤️

**Pull request merged**